test/extended/image_ecosystem/mysql_replica.go

```diff
@@ -187,36 +187,51 @@ func replicationTestFactory(oc *exutil.CLI, tc testCase) func() {
 			err = oc.Run("scale").Args("dc", "mysql-slave", "--replicas=4").Execute()
 			o.Expect(err).NotTo(o.HaveOccurred())
 			assertReplicationIsWorking("mysql-master-2", "mysql-slave-1", 4)
-*/
-/*	}
+		*/
+	}
 }
 
 var _ = g.Describe("[image_ecosystem][mysql][Slow] openshift mysql replication", func() {
 	defer g.GinkgoRecover()
 
-	var oc = exutil.NewCLI("mysql-replication", exutil.KubeConfigPath())
+	var oc = exutil.NewCLI("mysql-replication", exutil.KubeConfigPath()).Verbose()
```

Comment:
is the verbose intentional or leftover debug?

Reply:
leftover, i will remove

Reply:
i accidentally re-added the test as well ... still need to sort our the delete / re-add slave problem